### ISSUE TITLE
please remove template, as we would like to re-publish from a new repo

### ISSUE THREAD

**Issue by az-qbradley** [typesafehub/activator#1118]
Please remove the following template, because we would like to re-publish from another repo with the same application name, the permission wouldn't allow us otherwise.

https://www.lightbend.com/activator/template/squbs-sample

Thanks a lot,
Qian

**Comment by corruptmemory** [comment]
@az-qbradley, I've removed the template in question.  It may take some time for it to be purged from the template index.

**Comment by az-qbradley** [comment]
thanks a lot!

**Comment by az-qbradley** [comment]
It had gone from searching index, however, I am still not able to push to the same name with another repo, is that expected?

~~~~ I am seeing following error trying to re-publish from https://github.com/paypal/activator-squbs-scala.git (different repo, with same template name)
(snippet)
This template failed to publish. Please try again (after correcting any issues).
You do not have permission to publish 'squbs-sample'; try another name?

thanks a lot

**Comment by corruptmemory** [comment]
Right, it could be a long as a day before you can get your update taken.

**Comment by az-qbradley** [comment]
got it, thanks again.

**Comment by corruptmemory** [comment]
@az-qbradley: Saw your comment on Gitter.  Sorry you are still having problems.  I can fix this evening my time (US EDT) and I'll let you know when it's ready.

**Comment by corruptmemory** [comment]
Hrm.  Ok so some time lag on my part.  It appears that you republished the template under another name.

**Comment by az-qbradley** [comment]
Yes, we decided to publish under another name. Thanks a lot again for the follow-up.
